eth/headers.go

```diff
@@ -227,7 +227,8 @@ func (ehdr EthernetHeader) AssertType() EtherType { return EtherType(ehdr.SizeOr
 
 // Put marshals the ethernet frame onto buf. buf needs to be 14 bytes in length or Put panics.
 func (ehdr *EthernetHeader) Put(buf []byte) {
-	_ = buf[13]
+	_ = buf[13] 
```

Comment:
Please discard these changes, I don't see a diff.

Reply:
My fault - extra whitespace, I had commented line 230 - explaining the bounds check - but left a trailing space when removing it